HW5.py

```diff
@@ -0,0 +1,20 @@
+import csv, json
```

Comment:
Imports should be **on separate lines** according to PEP8
By the way, the general recommendation is that imports should be grouped in the following order:

1. Standard library imports.
2. Related third party imports.
3. Local application/library specific imports.

You should put a blank line between each group of imports. Each item in the group should be sorted alphabetically.